introduction.md

```diff
@@ -65,11 +66,55 @@ Forwarded: for=82.115.151.177; host=example.com; proto=https
 |     Required     | Header              | Description                                                                                                                                                                                                                                                                    |
 | :--------------: | :------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | {% icon check %}︎ | **`Content-Type`**  | The [content type][content-type] of the body of the HTTP request. Usually set to `application/json`.                                                                                                                                                                           |
-| {% icon check %}︎ | **`Accept`**        | The [content type][content-type] accepted by the client. Usually set to `application/json` and `application/problem+json` so both regular responses as well as errors can be received properly.                                                                                |
 | {% icon check %}︎ | **`Authorization`** | The OAuth 2 Access Token is generated in [Swedbank Pay Admin][admin].                                                                                                                                                                                                          |
+| {% icon check %}︎ | **`User-Agent`**    | The [user agent][user-agent] of the HTTP client making the HTTP request. Should be set to identify the system performing requests towards Swedbank Pay. The value submitted here will be returned in the response field `initiatingSystemUserAgent`.                           |
```

Comment:
Dang, I shouldn't have made this required. Can you please remove the required check icon?

```suggestion
|                  | **`User-Agent`**    | The [user agent][user-agent] of the HTTP client making the HTTP request. Should be set to identify the system performing requests towards Swedbank Pay. The value submitted here will be returned in the response field `initiatingSystemUserAgent`.                           |
```